app/lib/ui/screens/profile/profile_view.dart

```diff
@@ -79,29 +79,38 @@ class ProfileView extends StatelessWidget {
         ),
       );
 
-  Widget _buildPasswordInput(ProfileModelData data) => CustomTextFormField(
-        onChanged: controller.updatePassword,
-        initialText: data.user.name,
-        label: 'Password',
-        hint: 'Edit your password',
-        icon: Icons.lock_outline,
-        isPasswordField: true,
+  Widget _buildPasswordInput(ProfileModelData data) => Builder(
```

Comment:
why wrap it in a builder?

Reply:
I need the`BuildContext` for the focus scope